src/App.vue

```diff
@@ -1,15 +1,29 @@
 <template>
-  <img class="logo" alt="League of Legends logo" src="./assets/logo.png">
-  <ShowcaseTitle msg="LoL Champions Showcase"/>
+  <div id="#app">
```

Comment:
the hashtag is not needed, did you put it thinking in a specific purpose?

Reply:
Actually, I just kept the id that comes on the default boilerplate. I'll fix it.

Reply:
Done!